packages/web3-eth-abi/src/utils.ts

```diff
@@ -32,6 +33,13 @@ export const isAbiFunctionFragment = (item: unknown): item is JsonAbiFunctionFra
 	(item as { type: string }).type !== undefined &&
 	(item as { type: string }).type === 'function';
 
+export const isAbiConstructorFragment = (item: unknown): item is JsonAbiConstructorFragment =>
```

Comment:
Might want to move these `is_` methods to a `validation.ts` file to match `web3-eth` and `web3-utils`

Reply:
Will look into doing this in next PR.